src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <div className="Person">
```

Comment:
suggest to destructure person

```suggestion
export const Person = ({ person }) => {
const { // destructure here } = person;
return (
  <div className="Person">
```